invenio_rdm_records/services/config.py

```diff
@@ -95,20 +96,29 @@ class RDMRecordServiceConfig(RecordServiceConfig, RecordConfigMixin):
     schema_secret_link = SecretLink
 
     # Permission policy
-    permission_policy_cls = RDMRecordPermissionPolicy
+    permission_policy_cls = FromConfig(
+        "RDM_PERMISSION_POLICY", default=RDMRecordPermissionPolicy,
```

Comment:
Comment: Should the default only be defined one place?

Right now it's in ``config.py`` and here?